ultra/ultra/scenarios/interface.py

```diff
@@ -65,7 +65,7 @@
         "--pool-dir",
         help="directory for locating maps",
         type=str,
-        default=None,
+        default="pool/experiment_pool",
```

Comment:
I think this has to be relative to the directory we run `interface.py` from (the main `ultra/` directory). Or have you tested it? Does this work?

Reply:
Yes, so the path for the pool directory is relative in terms of the scenarios folder. I have used a previously defined `scenarios_dir` path (as shown below) to join the `pool_dir``
https://github.com/huawei-noah/SMARTS/blob/125da655b4e28a5ebd00dd1283442d0828d6db09/ultra/ultra/scenarios/generate_scenarios.py#L707-L713

Also inside of `tests/test_analysis` we use the `pool_dir` argument to explicitly define the `pool/analysis_pool` path. The test does pass as well
https://github.com/huawei-noah/SMARTS/blob/125da655b4e28a5ebd00dd1283442d0828d6db09/ultra/tests/test_analysis.py#L50-L58

Reply:
Ahh yes, I see.

In order to be consistent with where we are defining our directories from, perhaps we should make this relative to the base ULTRA folder: `pool/experiment_pool` -> `ultra/scenarios/pool/experiment_pool`. For example, the `--root-dir` argument of `interface.py` has a default value relative to the base ULTRA directory (`ultra/scenarios`).

Reply:
I do agree with that change. I have made those changes in the commits below

Reply:
I do agree with that change. I have made those changes in the commits below

Reply:
Oh awesome, thanks! Just made a small change to your change.